todo_app/app.py

```diff
@@ -1,15 +1,46 @@
-from flask import Flask
+from todo_app.data import session_items
+from flask import Flask, render_template
+from flask.globals import request, session
 
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
-@app.route('/')
-def index():
-    return 'Hello World!'
+@app.route('/', methods=["GET"])
+def index_get():
+    items =  session_items.get_items()
+    return render_template('index.html',items=items)
 
 
+@app.route('/', methods=["POST"])
+def edit_item():
```

Comment:
There's quite a bit of logic in this function, it would be nice to have the different actions hit different endpoints (e.g. `/complete`, `/delete`, `/reset`) rather than combining them all here